File: write_files.mjs
import { readFileSync, writeFileSync, mkdirSync } from 'fs';
import { dirname, join } from 'path';
import { fileURLToPath } from 'url';

const dir = dirname(fileURLToPath(import.meta.url));

const files = [
  { src: 'Header_final.tsx', dst: join('app', 'components', 'Header.tsx') },
  { src: 'page_final.tsx',   dst: join('app', 'page.tsx') },
  { src: 'billing_final.tsx', dst: join('app', 'billing', 'page.tsx') },
];

for (const { src, dst } of files) {
  const content = readFileSync(join(dir, src), 'utf8');
  mkdirSync(dirname(dst), { recursive: true });
  writeFileSync(dst, content, 'utf8');
  console.log('OK:', dst);
}
console.log('Hotovo!');

File: write_files.py
import os, shutil

script_dir = os.path.dirname(os.path.abspath(__file__))
project_dir = script_dir  # skript je v rootu projektu

files = {
    "Header_final.tsx": os.path.join("app", "components", "Header.tsx"),
    "page_final.tsx": os.path.join("app", "page.tsx"),
    "billing_final.tsx": os.path.join("app", "billing", "page.tsx"),
}

for src, dst in files.items():
    src_path = os.path.join(script_dir, src)
    dst_path = os.path.join(project_dir, dst)
    if os.path.exists(src_path):
        os.makedirs(os.path.dirname(dst_path), exist_ok=True)
        shutil.copy2(src_path, dst_path)
        print(f"OK: {dst}")
    else:
        print(f"CHYBA: {src} nenalezen")

print("Hotovo!")
